front/app/api/campaigns/useCampaigns.ts

```diff
@@ -0,0 +1,33 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import campaignsKeys from './keys';
+import { ICampaignsData, QueryParameters, CampaignsKeys } from './types';
+
+const fetchCampaigns = (filters: QueryParameters) => {
+  const {
+    campaignNames: campaign_names,
+    withoutCampaignNames: without_campaign_names,
+    pageNumber,
+    pageSize,
+  } = filters;
+  return fetcher<ICampaignsData>({
+    path: '/campaigns',
+    action: 'get',
+    queryParams: {
+      campaign_names,
+      without_campaign_names,
+      'page[number]': pageNumber,
+      'page[size]': pageSize,
```

Comment:
I think we want fallback values here, like we did in GetCampaigns. Also see other hooks (e.g. useAreas).